src/Cake.Core/Tooling/ToolSettings.cs

```diff
@@ -39,6 +39,9 @@ public class ToolSettings
         ///     new NuGetSourcesSettings { UserName = "user", Password = "incorrect",
         ///     ArgumentCustomization = args=&gt;args.Append("-StorePasswordInClearText")
         /// });
+        /// // or with multiple arguments:
+        /// MSTest(pathPattern, new MSTestSettings() { ArgumentCustomization = args =&gt; args.Append("/detail:errormessage")
+		///	                                                                                  .Append("/resultsfile:TestResults.trx") });
```

Comment:
We use spaces not tabs
![image](https://cloud.githubusercontent.com/assets/1647294/16723701/dcef51d2-474e-11e6-982c-c5971ce9cbd7.png)
so that's likely the stylecop issue.